docs/patterns/triggers/triggers.mdx

```diff
@@ -10,8 +10,14 @@ You can enable a trigger by specifying the trigger enum, available on the [dashb
 <Tabs>
 <Tab title="CLI">
 <Steps>
+<Step title="Get Config Parameters">
+View the required configuration parameters for enabling a trigger by using the show command. For example, this will display what parameters are needed for Github's new star event trigger:
+```Bash
+composio triggers show GITHUB_STAR_ADDED_EVENT
+```
+</Step>
 <Step title="Enable Trigger">
-You need to pass the trigger enum to enable a trigger.
+You need to pass the trigger enum to enable a trigger & you'll be prompted to enter the required parameters.
```

Comment:
Consider using 'and' instead of '&' for better readability and consistency with technical documentation standards.